### PR TITLE
Version 2 of the finserv month lp

### PR DESCRIPTION
**Pull request by caldav**
## Done

* New version of finserv lp for A/B test at [`/engage/financial-services-month2`](http://www.ubuntu.com-pr-4078.run.demo.haus/engage/financial-services-month2)

### CODE REVIEW

File: templates/engage/financial-services-month2.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+{% block content %}
+<section class="p-strip--dark is-deep p-takeover--financial-services">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-7">
+      <h1 class="p-takeover__title">Canonical presents<br> Financial Services month</h1>
+      <p>Filmed at the London Stock exchange with representatives from Canonical, 451 Research, IBM, and a former Deutsche Bank storage CTO, Canonical took a deep dive look in to the rise of multi-cloud in the financial services industry.</p>
+      <p>Join us for insights around emerging technologies including AI/ML, Blockchain and Containers in FinTech and contact us to discuss your business case.</p>
+    </div>
+    <div class="col-1">&nbsp;</div>
+    <div class="col-5 u-align--center u-vertically-center">
```
Comment (Caleb-Ellis) [inline templates/engage/financial-services-month2.html]: You can add the class "prefix-1" here to have an empty col-1 space to the left (or you could add suffix-1 to the col-7). It's not crazy important here, but generally we should try to maintain a 12 column grid.

File: templates/engage/financial-services-month2.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      <h2>Ambitions and challenges mark financial services plans for cloud </h2>
+      <p>Referencing 451 Research’s Voice of the Enterprise data, Liam Eagle, Research Manager at 451 Research, looks at the adoption trends of cloud within the financial services sector where investment in the promise of cloud is strong, but cautious in how central it is to their IT operations. </p>
+      <p>Watch this webinar to discover: </p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">How financial services are adopting cloud services</li>
+        <li class="p-list__item is-ticked">What advanced features are most commonly being adopted in financial services</li>
+        <li class="p-list__item is-ticked">What a multi-cloud future for most businesses looks like</li>
+      </ul>
+      <p><a href="https://www.brighttalk.com/webcast/6793/334389?utm_source={{utm_source}}&utm_campaign=FY19_Cloud_Finance_WBN_451Finance" class="p-button--positive" >Watch on-demand webinar</a>&nbsp;<a href="http://0.0.0.0:8001/contact-us/form?product=generic-contact-us" class="p-button--neutral" >Discuss your business case</a></p>
```
Comment (Caleb-Ellis) [inline templates/engage/financial-services-month2.html]: Not sure about the href here... :P

File: templates/engage/financial-services-month2.html [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+      <p>Watch this webinar for:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Insights into the biggest challenges facing both the financial sector and the cloud landscape</li>
+        <li class="p-list__item is-ticked">A look at Canonical’s presence in the financial industry</li>
+        <li class="p-list__item is-ticked">Detailed use cases across Canonical’s product portfolio</li>
+      </ul>
+      <p><a href="https://www.brighttalk.com/webcast/6793/334391?utm_source={{utm_source}}&utm_campaign=FY19_Cloud_Finance_WBN_DevOpsInFS" class="p-button--positive" >Watch on-demand webinar</a>&nbsp;<a href="http://0.0.0.0:8001/contact-us/form?product=generic-contact-us" class="p-button--neutral" >Discuss your business case</a></p>
```
Comment (Caleb-Ellis) [inline templates/engage/financial-services-month2.html]: href

File: templates/engage/financial-services-month2.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+      <p>Find out why these factors are leading financial services organisations to rethink their infrastructures - leading to the adoption of a multi-cloud strategy. In addition, you'll learn about:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Best practices for starting a multi-cloud journey</li>
+        <li class="p-list__item is-ticked">Dealing with regulatory, security, and geographical challenges</li>
+        <li class="p-list__item is-ticked">What the tangible benefits of multi-cloud are</li>
+      </ul>
+      <p><a href="https://www.brighttalk.com/webcast/6793/334392?utm_source={{utm_source}}&utm_campaign=FY19_Cloud_Finance_WBN_MulticloudFS" class="p-button--positive">Watch on-demand webinar</a>&nbsp;<a href="http://0.0.0.0:8001/contact-us/form?product=generic-contact-us" class="p-button--neutral" >Discuss your business case</a></p>
```
Comment (Caleb-Ellis) [inline templates/engage/financial-services-month2.html]: href

File: templates/engage/financial-services-month2.html [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+      <h2>Blockchain, AI/ML and Containers - The impact new technologies will have on Financial Services</h2>
+      <p>Huge technological breakthroughs have been made over the past few years, with the rise of Artificial Intelligence, blockchain, containers, Kubernetes. Barely a week seems to go by without a new game-changing technology. There’s increasing pressure on financial services to change, but how do they start?</p>
+      <p>Join this webinar for information on: </p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">The challenges & benefits of blockchain, AI/ML, and containers in the financial industry</li>
+        <li class="p-list__item is-ticked">How businesses can get aligned with these new technologies</li>
+        <li class="p-list__item is-ticked">Examples of successful deployments</li>
+      </ul>
+      <p><a href="https://www.brighttalk.com/webcast/6793/334394?utm_source={{utm_source}}&utm_campaign=FY19_Cloud_Finance_WBN_BlockainAIContainers" class="p-button--positive">Register for webinar</a>&nbsp;<a href="http://0.0.0.0:8001/contact-us/form?product=generic-contact-us" class="p-button--neutral" >Discuss your business case</a></p>
```
Comment (Caleb-Ellis) [inline templates/engage/financial-services-month2.html]: href

**Comment by frankiegranato** [comment]
I guess you are using the four card row because you want to get all the webinars info "above the fold". This is not an approach I would necessarily support but if you insist on doing it, then I have the following suggestions:

- This four card row is not a pattern we use (we only use 3 card or 2 card rows). See this page as an example: https://www.ubuntu.com/kubernetes/install
- Also, the styling (with the coloured strip) is not appropriate for this use-case. We only use that style for blog & tutorial listings. 
- Finally, I would suggest you add an anchor link from the cards to the more in-depth sections below otherwise it seems pointless to me to include the below content at all. 
Also, I should add that we are really really busy this iteration so Ana doesn't have a lot of time to help you redesign this page so I suggest you stick to the vanilla patterns.

**Comment by frankiegranato** [comment]
@caldav I just re-read my last comment and it sounded a bit harsh - I'm sorry - I wrote it in a rush and I didn't mean to sound unfriendly :-) Please do let me know if I can help any further.

**Comment by anasereijo** [comment]
Hi @caldav I agree with Frankie.
We only should have 2 or 3 cards per row. At the moment the content looks very small and crowded.
Maybe we can have the first 3 as cards and the 4th card can be a strip under that section?
Can we also remove the coloured labels?

**Comment by anasereijo** [comment]
@caldav what is happening to this? let me know if you need anything

**Comment by pmahnke** [comment]
closing for now as @caldav 	doesn't have time to fix-up... leaving branch though

**Pull request closed without merging**